include/wx/imagwebp.h

```diff
@@ -0,0 +1,72 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/imagwebp.h
+// Purpose:     wxImage WebP handler
+// Author:      Hermann Höhne
+// Created:     2024-03-08
+// Copyright:   (c) Hermann Höhne
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_IMAGWEBP_H_
+#define _WX_IMAGWEBP_H_
+
+#include "wx/image.h"
+#include "wx/colour.h"
+
+//-----------------------------------------------------------------------------
+// wxWEBPHandler
+//-----------------------------------------------------------------------------
+
+#if wxUSE_LIBWEBP
+
+namespace wxWebPImageOptions
+{
+    enum : int
+    {
+        Undefined = 0,
+        Lossy = 1,
+        Lossless = 2
+    };
+}
+
+#define wxIMAGE_OPTION_WEBP_QUALITY  "WebPQuality"
+#define wxIMAGE_OPTION_WEBP_LOSSLESS "WebPLossless"
+
+
+struct wxWebPAnimationFrame
+{
+    wxImage image;
+    wxColour bgColour;
+    int duration = 0;
+};
+
+
+class WXDLLIMPEXP_CORE wxWEBPHandler : public wxImageHandler
+{
+public:
+    inline wxWEBPHandler()
```

Comment:
Very minor, but we typically don't use `inline` expicitly

```suggestion
    wxWEBPHandler()
```